### PR TITLE
Update _etl_lead_attribution.md

### PR DESCRIPTION
**Pull request by ihtnayaj**

### CODE REVIEW

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
 
-When creating an ETL Lead Attribution Report, provide the following fields in `report_params`:
-
-| Param | Function |
+|Method|URI Format|
 |---|---|
-|`gmaid`|Global Master Advertiser ID.  Only events for a single advertiser can be returned in a single report|
-|`start_date`|Restricts the results to those occurring on or after this date.|
-|`end_date`|Restricts the results to those occurring on or before this date.|
+|GET|/client_reports/unified_display/[gmaid]?[query_params]|When creating an ETL Lead Attribution Report, provide the following fields in `report_params`:
```
Comment (brianroth-rl) [inline source/includes/etl_reports/_etl_lead_attribution.md:8]: To create an ETL report, a POST is required.  This URL is also incorrect.

I also feel that the fact that there is an asynchronous request/response process used to generate this report isn't clear at all in this documentation (this is more of a comment on earlier work).  The documentation must stress how you create the job and then need to poll an API endpoint until the job is complete.  No mention of the GET portion of the request to retrieve the status and the actual file is made here.

**Review by brianroth-rl — CHANGES_REQUESTED** [review]

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 2]
```diff
@@ -1,17 +1,36 @@
-### ETL Lead Attribution Report
+### **ETL Lead Attribution Report**
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md]: updated

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 3]
```diff
@@ -1,17 +1,36 @@
...
+<a name="Lead Attribution report"></a>
 
-#### Report Params
+### Resource Overview&nbsp;
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md]: updated

File: source/includes/etl_reports/_etl_lead_attribution.md (hunk 1 above)
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md:8]: I will rewrite it.

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
-
-| Param | Function |
+|Method|URI Format|
 |---|---|
-|`gmaid`|Global Master Advertiser ID.  Only events for a single advertiser can be returned in a single report|
-|`start_date`|Restricts the results to those occurring on or after this date.|
-|`end_date`|Restricts the results to those occurring on or before this date.|
+|GET|/client_reports/unified_display/[gmaid]?[query_params]|When creating an ETL Lead Attribution Report, provide the following fields in `report_params`:
+
+### Parameters&nbsp;
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md]: updated

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 5]
```diff
@@ -1,17 +1,36 @@
...
+|Parameter|Required|Description|
+|---|---|---|
+|`gmaid`|Yes|Global Master Advertiser ID.  Only events for a single advertiser can be returned in a single report|
+|`start_date`|Yes|Restricts the results to those occurring on or after this date.|
+|`end_date`|Yes|Restricts the results to those occurring on or before this date.|
 
 All params are required.  Once the job has completed, a call to the status endpoint will yeild a signed_url to a csv file, which can be downloaded.
 
+
+### Response Data Details&nbsp;
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md]: updated

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 6]
```diff
@@ -28,6 +47,7 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
 }'
 ```
 
+### Response Data Details&nbsp;
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md]: updated

**Review by brianroth-rl — CHANGES_REQUESTED** [review]

File: source/index.html.md [hunk 7]
```diff
@@ -65,6 +65,8 @@ includes:
   - fpd
   - capture_external_event_fpd
   - capture_external_event_chat
+  - etl_reports
+  - etl_lead-attribution
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/index.html.md]: updated

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 8]
```diff
@@ -28,8 +47,76 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
 }'
 ```
 
-> The format of the resulting csv file is:
+### Response Data Details;
+> The resulting csv file will contain the following
+
+Field | Type  | Description
+--------- | -------- |--------
+
```
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md:61]: updated

File: source/index.html.md [hunk 9]
```diff
@@ -65,6 +65,8 @@ includes:
   - fpd
   - capture_external_event_fpd
   - capture_external_event_chat
+  - etl_reports
+   - etl_reports/etl_lead_attribution
```
Comment (brianroth-rl) [inline source/index.html.md:69]: This isn't right, it results in in an error
```
Error: Could not locate partial: includes/etl_reports - etl_reports/_etl_lead_attribution
```
You have a space character before the -, remove it and it works

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 10]
```diff
@@ -1,16 +1,41 @@
 ### ETL Lead Attribution Report
+<a name="Lead Attribution report"></a>
 
-#### Report Params
+### Resource Overview;
```
Comment (brianroth-rl) [inline source/includes/etl_reports/_etl_lead_attribution.md:4]: The ; character is probably not wanted

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 11]
```diff
@@ -28,8 +53,76 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
 }'
 ```
 
-> The format of the resulting csv file is:
+### Response Data Details;
```
Comment (brianroth-rl) [inline source/includes/etl_reports/_etl_lead_attribution.md:56]: This was pointed out earlier and marked as resolved, but the ; character is still present.

File: source/includes/etl_reports/_etl_lead_attribution.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (brianroth-rl) [reply source/includes/etl_reports/_etl_lead_attribution.md:61]: It's still broken

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 12]
```diff
@@ -28,8 +53,76 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
...
+|email| String | no | email of the contact|
+|occurrence_time| Datetime | The date and time which this contact interaction occurred|
+|chat_transcript|String|Transcript of the chat|	
+|call_duration|yes|number|The duration of the call in seconds|
+|call_recording_url|yes|string|The URL address to an audio recording of the call|
+|phone_work|yes|string|Normalized phone number of the lead|
+
+**Event Sub Type**
+Describes the type of event
+| Field Name |
```
Comment (brianroth-rl) [inline source/includes/etl_reports/_etl_lead_attribution.md:89]: To get this table to render, a blank line needs to be added after the word event and before the | character.

Also, this table looks strange.  It only has one column labeled field name.  Should this be a table?  If it stays a table, I would expect a second column with a field description.

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 13]
```diff
@@ -28,8 +53,76 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
...
+|Cvt |
+|ChatInitiated |
+|Consent |
+|Facebook |
+|Unavailable |
+
+
+**Referrer Source**
+Describes the traffic source of  the visitor.
+| Field Name | Description |
```
Comment (brianroth-rl) [inline source/includes/etl_reports/_etl_lead_attribution.md:106]: To get this table to render, a blank line needs to be added after the word visitor. and before the | character.

File: source/includes/etl_reports/_etl_lead_attribution.md [hunk 14]
```diff
@@ -28,8 +53,76 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
...
+|---|---|
+|Organic|Visitor's came to the site organically|
+|Paid|Visitor's came to the site through a LOCALiQ campaign link|
+|NonRLPaid|Visitor's came to the site through a paid link but is not LOCALiQ|
+|Unavailable|Cannot be determined|
+
+**Referrer Type**
+Describes the type of domain the visitor came from.
+| Field Name |Description |
```
Comment (brianroth-rl) [inline source/includes/etl_reports/_etl_lead_attribution.md:115]: To get this table to render, a blank line needs to be added after the word from. and before the | character.

**Review by brianroth-rl — CHANGES_REQUESTED** [review]

File: source/includes/etl_reports/_etl_lead_attribution.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (ihtnayaj) [reply source/includes/etl_reports/_etl_lead_attribution.md]: updated and looked good in preview mode (fingers crossed)

**Pull request merged**